pkg/keys/keys.go

```diff
@@ -60,50 +61,55 @@ type EcdsaPublic struct {
 	PublicKey data.HexBytes `json:"public"`
 }
 
-func toPubKey(pemBytes []byte) (*ecdsa.PublicKey, error) {
-	if pemBytes == nil {
-		return nil, errors.New("failed to get public key")
-	}
-	block, rest := pem.Decode(pemBytes)
-	if len(rest) != 0 {
-		return nil, errors.New("failed to parse public key PEM block")
-	}
-	pubkey, err := x509.ParsePKIXPublicKey(block.Bytes)
+func ToCert(pemBytes []byte) (*x509.Certificate, error) {
```

Comment:
I see `ToCert` and `toCert`, can we delete the private one?

Reply:
Yeah, I think so, one was for files and one was for bytes, but keeping the bytes is easier for testing